manifests/pe_metric.pp

```diff
@@ -10,6 +10,9 @@
   String                    $metric_script_file = 'tk_metrics',
   Array[Hash]               $additional_metrics = [],
   Boolean                   $ssl                = true,
+  Boolean                   $use_splunk_hec     = false,
```

Comment:
I'm kind of thinking we shouldn't special case splunk here.  What about having an `enum['graphite','influxdb','splunk'] export_to` parameter and get rid of metrics server type inside of metrics_server_info.